py/tests/string.py

```diff
@@ -99,6 +99,9 @@ class C():
 asc="hello"
 uni="£100世界𠜎" # 1,2,3,4 byte unicode characters
 
+doc="split"
+assert 4 == len(list("0,1,2,4".split(",")))
```

Comment:
I think `assert ["0","1","2","4"] == "0,1,2,4".split(",")` would be better here.  A few more tests would be good too, like for empty string.